README.md

```diff
@@ -49,7 +50,43 @@ Similarly, [calorimetric.json](https://github.com/building-envelope-data/api/blo
 
 With you web browser, you can search our [wiki](https://github.com/building-envelope-data/api/wiki), the [issues](https://github.com/building-envelope-data/api/issues) and [pull requests](https://github.com/building-envelope-data/api/pulls) and contribute to them.
 
-In order to browse the code conveniently, you should first [clone the repository](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) and then use a text editor, for example [Visual Studio Code](https://code.visualstudio.com/).
+### In your browser
+
+In order to browse the code conveniently, you can click on `Code` and then `+` to start a new codespace.
+
+![Start a new codespace](docs/images/open_codespace.png)
+
+If you are developing this repository further, you can use the following steps to test and format your contributions:
+
+1. Use the terminal at the bottom right of the codespace to prepare your environment with 
+   ```shell
+   cp ./.env.sample ./.env
+   ```
+1. Run
```

Comment:
Ab hier ist es dasselbe wie `With Docker` (mit Ausnahme des ersten Schritts) und hier wird ja auch ueber `make shell` tatsaechlich Docker verwendet. Irgendwie faende ich es gut einfach darauf zu verweisen. Mir faellt da gerade nut kein guter Satz dazu ein.

Reply:
I have referred to section `With Docker` and kept a simple example for beginners in [faf9a9b](https://github.com/building-envelope-data/api/pull/336/commits/faf9a9be4fc2e6a65526cc289c3b42f81a0cfc06).